src/App.tsx

```diff
@@ -1,73 +1,33 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
-import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import { Dropdown } from './dropdown';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selectedPerson, setSelectedPerson] = useState<Person>();
+
+  const onPersonSelected = (person?: Person) => {
+    setSelectedPerson(person);
+  };
+
+  const chosenPerson = () => {
+    return selectedPerson
+      ? `${selectedPerson.name} (${selectedPerson.born} - ${selectedPerson.died})`
+      : 'No selected person';
+  };
+
+  const handleInputChanged = () => {
```

Comment:
If it was done because of this:
![image](https://github.com/mate-academy/react_autocomplete/assets/86923089/0d80ad6b-5ea1-41a6-9189-574582eb70b0)

it's the wrong solution. You should hide dropdown suggestions:
![image](https://github.com/mate-academy/react_autocomplete/assets/86923089/159ad6a2-5b9f-4931-97e1-b90f19b2a759)